config/config.go

```diff
@@ -64,14 +64,14 @@ func (e MissingEntityError) Error() string {
 	return fmt.Sprintf("could not find the referenced (%s) %s", e.Entity, e.Name)
 }
 
-// ConfigError is an error for invalid configuration that includes what entity errored.
-type ConfigError struct {
+// Error is an error for invalid configuration that includes what entity errored.
+type Error struct {
```

Comment:
I think "ConfigError" or "EntityError" is better. Easy to typo or confuse `error` with `Error`

Reply:
Issue is that this gets written as `config.ConfigError` outside the package, which stutters, and why the linter will complain about it.

Reply:
That's a good point. "ValidationError"?

Reply:
ValidationError SGTM, updated!